### PR TITLE
Fixing R3023 Operation API Implmentation ARM Swagger error

### PR DESCRIPTION
**Pull request by vakkinep**
Validated with autorest --azure-validator --input-file=dedicatedhsm.json

Issue: https://msazure.visualstudio.com/One/_workitems/edit/3680852
PBI: https://msazure.visualstudio.com/One/_workitems/edit/5071242

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by vakkinep** [comment]
Issue: https://msazure.visualstudio.com/Unified%20Platform%20KPIs/_workitems/edit/3680852

**Comment by fengzhou-msft** [comment]
Please fix the ModelValidation error.

File: specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2018-10-31-preview/dedicatedhsm.json [hunk 1]
```diff
@@ -392,6 +417,45 @@
...
+              "type": "string"
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed.",
+              "type": "string"
+            },
+            "operation": {
+              "description": "Operation type: Read, write, delete, etc.",
+              "type": "string"
+            }
```
Comment (anthony-c-martin) [inline specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2018-10-31-preview/dedicatedhsm.json]: This is missing the `description` property as defined in the contract [here](https://github.com/Azure/azure-resource-manager-rpc/blob/fbb33d93f0b2cc58ef345c3a430766632ad23936/v1.0/proxy-api-reference.md#exposing-available-operations). Please could you add this?

**Review by anthony-c-martin — CHANGES_REQUESTED** [review]

**Comment by anthony-c-martin** [comment]
@vakkinep One comment - please take a look.

File: specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2018-10-31-preview/dedicatedhsm.json (hunk 1 above)
Comment (vakkinep) [reply specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2018-10-31-preview/dedicatedhsm.json:442]: Hi @anthony-c-martin , the new changes are here:

**Comment by KrisBash** [comment]
@vakkinep  - will you abandon this PR? It looks like you added changes to a new PR

**Pull request closed without merging**